### DISCUSSION TITLE
[Transfer Learning] Import a pretrained FNN to a PFNN and freeze the weights of the pretrained FNN

### DISCUSSION THREAD

**Discussion by haison19952013** [lululxvi/deepxde#726] [Q&A]
Hi Dr. LuLu and the great community of deepXDE,
- Recently, I have faced a problem that needs to apply transfer learning for PINN. My questions are:
1. Is there any way to import my pre-trained FNN (Inputs size: 3, Outputs size: 3) to a PFNN (ie., a system of 2 FNN with Inputs size: 3, and Outputs size: 3. So, inputs size: 6 in total and outputs size: 6 in total) and then freeze the pre-trained FNN?
2. Is there any way to change the input size and output size of a  pre-trained FNN and continue the training?

**Comment by lululxvi** [comment]
DeepXDE doesn't support these. You have to modify the source code.
